config/manager/kustomization.yaml

```diff
@@ -19,7 +19,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/kiegroup/kogito-serverless-operator
+  newName: quay.io/kiegroup/kogito-serverless-operator-nightly
   newTag: 2.0.0-snapshot
```

Comment:
shouldn't be latest?

Reply:
`latest` won't be accepted by the operator-sdk framework as a correct version ;)